content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -188,6 +188,27 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
   const router = useRouter();
   const [stopNumber, setStopNumber] = useState(stopNumberServerSide);
   const [currentStop, setCurrentStop] = useState(currentStopServerSide);
+  const [headerEl, setHeaderEl] = useState<HTMLElement>();
```

Comment:
I know this was like this before but could be of type `HTMLHeadElement` to be even more specific?

Reply:
I wondered this, and went to add it as an `HTMLHeaderElement`, but that didn't exist (my options were `HTMLHeadElement` or `HTMLHeadingElement`). Seems like if an element doesn't have any additional requirements from the default (which I guess this doesn't) then it's a plain old HTMLElement (found [here](https://stackoverflow.com/questions/65721709/why-is-there-no-htmlsectionelement-and-no-htmlarticleelement-in-javascript))